src/services/AccessTokenRequestProcessor.ts

```diff
@@ -52,10 +52,12 @@ export class AccessTokenRequestProcessor {
 			let session: ISessionItem | undefined;
 			try {
 				session = await this.f2fService.getSessionByAuthorizationCode(requestPayload.code);
-				this.logger.info({ message: "Found Session: " + JSON.stringify(session) });
 				if (!session) {
+					this.logger.info(`No session found by authorization code: : ${requestPayload.code}`, { messageCode: MessageCodes.SESSION_NOT_FOUND });
 					return new Response(HttpCodesEnum.UNAUTHORIZED, `No session found by authorization code: ${requestPayload.code}`);
 				}
+				this.logger.info({ message: "Found Session" });
+				this.logger.appendKeys({ sessionId: session.sessionId });
```

Comment:
this ensures the info log contains the `sessionId` key
```suggestion
				this.logger.appendKeys({ sessionId: session.sessionId });
				this.logger.info({ message: "Found Session" });
```